Robust.Shared/Prototypes/PrototypeManager.ValidateFields.cs

```diff
@@ -63,7 +63,7 @@ private void ValidateStaticField(
         Dictionary<Type, HashSet<string>> prototypes)
     {
         DebugTools.Assert(field.IsStatic);
-        DebugTools.Assert(!field.HasCustomAttribute<DataFieldAttribute>(), "Datafields should not be static");
+        DebugTools.Assert(!field.HasCustomAttribute<DataFieldAttribute>(), $"Datafields should not be static. Field: {field.Name} in Type: {field.DeclaringType?.FullName}");
```

Comment:
```suggestion
        DebugTools.Assert(!field.HasCustomAttribute<DataFieldAttribute>(), $"Datafields should not be static. Field: {field.Name} in Type: {field.DeclaringType}");
```

Better to just let the full type be formatted, probably.

Reply:
👍